src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -107,14 +150,42 @@ public IDisposable On(string methodName, Type[] parameterTypes, Func<object[], o
             return new Subscription(invocationHandler, invocationList);
         }
 
+        /// <summary>
+        /// Invokes a streaming hub method using the specified method name, return type and arguments.
+        /// </summary>
+        /// <param name="methodName">The hub method name.</param>
+        /// <param name="returnType">The hub return type.</param>
+        /// <param name="args">The hub arguments.</param>
```

Comment:
`The arguments expected by the method`

Reply:
They aren't expected by the method. They're used to call the method.

Reply:
They're expected by the target method, is what I meant. "The arguments to pass to the method"?